todo_app/helpers/view_class.py

```diff
@@ -0,0 +1,25 @@
+from todo_app.helpers.item_class_helper import Card_Item
+
+class ViewModel:
+    def __init__(self, items: list[Card_Item]):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def done_items(self):
+        done_items =[]
+        for item in self.items:
+            if item.status == "Done":
+                done_items.append(item)
+        return done_items
```

Comment:
Nice and clean 👍 
Not necessarily a recommendation but I always think interesting to see a comparison with the list-comprehension approach which is arguably less readable, but more concise:
```suggestion
        return [item for item in self.items if item.status == "Done"]
```